### PR TITLE
improvement(cluster): keep instances on failure 72h

### PR DESCRIPTION
**Pull request by enaydanov**
Trello: https://trello.com/c/YxLby7KQ/1502-keep-alive-when-test-finish-and-set-to-keep-set-to-keep-for-72-hours

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [x] I added the relevant `backport` labels
- [x] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [x] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x] All new and existing unit tests passed (CI)
- [x] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by amoskong** [comment]
Can we send email notification when the time is expired?

**Comment by enaydanov** [comment]
> 
> 
> Can we send email notification when the time is expired?

@amoskong It's an interesting idea, but I think it's too big to be added in this PR. Please, add Trello card for it.

**Review by fruch — DISMISSED** [review]
LGTM

**Comment by fruch** [comment]
@enaydanov please address conflicts

File: sdcm/cluster.py [hunk 1]
```diff
@@ -439,7 +442,7 @@ def init(self) -> None:
         self.start_task_threads()
         self._init_port_mapping()
 
-        self.set_keep_alive()
+        self.set_keep_alive(until=datetime.utcnow() + timedelta(hours=KEEP_ALIVE_ON_FAILURE, minutes=TEST_DURATION))
```
Comment (fruch) [inline sdcm/cluster.py:445]: why add both 72 and the duration of the test ?

**Comment by bentsi** [comment]
please resolve conflicts

File: sdcm/cluster.py (hunk 1 above)
Comment (enaydanov) [reply sdcm/cluster.py:455]: Because it's 72h _after_ test complete.

**Review by amoskong — APPROVED** [review]
Looks good.

**Comment by fruch** [comment]
@enaydanov we still need this one ? The PR it depends on was merged long time ago

**Comment by bentsi** [comment]
closing this since Shlomi didn't approve the changes in cloud instance stopper repo

**Comment by bentsi** [comment]
@fruch the PR it depends was never merged https://github.com/scylladb/cloud-devops/pull/12

**Pull request closed without merging**